target/linux/ramips/image/mt76x8.mk

```diff
@@ -325,6 +325,15 @@ define Device/onion_omega2p
 endef
 TARGET_DEVICES += onion_omega2p
 
+define Device/qihoo_p2
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := Qihoo
+  DEVICE_MODEL := P2
+  DEVICE_PACKAGES := kmod-mt76x2 kmod-usb-ohci kmod-usb2 \
+  kmod-usb-ledtrig-usbport
```

Comment:
Single tab for hanging indent.

Reply:
I will try to make it work in the future, but maybe I need to suspend this PR for a while before that.